lib/url-prefixer.js

```diff
@@ -57,7 +57,7 @@ function urlPrefixer(config) {
     var chunk_remainder;
 
     return new Transform({
-      decodeStrings: false,
+      decodeStrings: yes,
```

Comment:
I think you mean `true`
```suggestion
      decodeStrings: true,
```

Reply:
Also, why make this change? It looks like it just causes causes extra work. `decodeStrings` means "convert each chunk of data from `string` to `Buffer`", but the urlPrefixer actually needs to work on `string`s, so it would just have to convert back.